src/Maestro/SubscriptionActorService/MaestroBarClient.cs

```diff
@@ -100,7 +100,18 @@ public Task<Build> GetLatestBuildAsync(string repoUri, int channelId)
 
         public Task<Subscription> GetSubscriptionAsync(Guid subscriptionId)
         {
-            throw new NotImplementedException();
+            var subscriptionObject = _context.Subscriptions.Where(s => s.Id.Equals(subscriptionId)).FirstOrDefault();
```

Comment:
Feels like this should be FirstOrDefaultAsync, which I think it a thing?  Also, `thing.Where(CODE).FirstOrDefault()` is the same as `thing.FirstOrDefault(CODE)`, except the second one is less brain thinking I have to do. :-)

Reply:
And then the method could be async and not have to do the Task.FromResult stuff. :-)